### ISSUE TITLE
Export built in password

### ISSUE THREAD

**Issue by vudoanbinhminh** [minbrowser/min#2114]
Is there any way to export the password?  I need to import it to other browsers (or password managers)

**Comment by delsallorenzo** [comment]
YES PLEASE. would have come in handy last week as I was setting up the new computer
